scripts/ci-deploy.sh

```diff
@@ -21,29 +22,48 @@ if [[ -z ${ANSIBLE_FOLDERS} ]]; then
   ANSIBLE_FOLDERS="infrastructure"
 fi
 
+if [[ -d ${cdk_app_dir} ]]; then
```

Comment:
`  -d `  would lead to unintentional mistakes. Remember the first repo Sebastian created? He copied the infra-basis repo then added his logic without removing the unnecessary ones. My proposal is to remove the definition of ` cdk_app_dir` , and default either CDK or Ansible case to ``` infrastructure` . When we speak about the folder layout for a simple repo (i.e. one infra + one biz app), having different naming of the infrastructure module folder does not bring additional benefit. CDK or Ansible (or something else in the future) is just a way of how the Infrastructure-as-Code is implemented.

Reply:
Yes, your proposal will make things simpler